astroquery/utils/tests/test_utils.py

```diff
@@ -27,10 +28,10 @@ class SimpleQueryClass(object):
     def query(self):
         """ docstring """
         if self is SimpleQueryClass:
-            print("Calling query as class method")
+            log.info("Calling query as class method")
```

Comment:
these can stay as prints as it doesn't matter in prints (pytest won't show the output unless there are test failures)